os/kernel/signal/sig_dispatch.c

```diff
@@ -440,10 +440,15 @@ int sig_dispatch(pid_t pid, FAR siginfo_t *info)
 
 	stcb = sched_gettcb(pid);
 	if (stcb) {
```

Comment:
please update description on top of this function about changes.

Reply:
Done.